main.go

```diff
@@ -35,9 +36,15 @@ var (
 	flagDebug       = flag.Bool("debug", false, "Enables debug messages")
 	flagInsecure    = flag.Bool("insecure", false, "Uses unencrypted connections when fetching images")
 	flagCompression = flag.String("compression", "gzip", "Type of compression to use; allowed values: gzip, none")
+	flagVersion     = flag.Bool("version", false, "Print version")
 )
 
-func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bool, flagInsecure bool, flagCompression string) error {
+func printVersion() {
+	fmt.Println("docker2aci version", docker2aci.Version)
+	fmt.Println("appc version", docker2aci.AppcVersion)
+}
+
+func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bool, flagInsecure bool, flagCompression string, flagVersion bool) error {
```

Comment:
I guess you really didn't want to use `*flagImage` or `*flagNoSquash` everytime. Maybe instead of shadowing global variables with local ones you could simply do:
```go
var (
    flagNoSquash bool = false
    flagCompression string = "gzip"
    …
)

func init() {
    flag.BoolVar(&flagNoSquash, "nosquash", "Blurb here")
    flag.StringVar(&flagCompression, "compression", "Blah blah blah")
    …
}
```
This way you can have the `runDocker2ACI` function with just one parameter.